### ISSUE TITLE
media:dump --strip should also exclude captcha generated images

### ISSUE THREAD

**Issue by erfanimani** [netz98/n98-magerun#448]

**Comment by pocallaghan** [comment]
I took a quick look at this, seems like the extra paths just need to be added to the bottom of config.yaml, is this correct? Paths I can think of that should probably be added are `css_secure`, `captcha`, `catalog/category/cache` and probably `wysiwyg/.thumbs`. Though due to core bugs, that last one might not work as well as it should.

**Comment by erfanimani** [comment]
You're right - it does like all that's needed is to add those paths to the config.yaml file. I'll see if I can give it a shot and maybe create a PR @pocallaghan

**Comment by Ig0r-M-magic42** [comment]
has this been fixed ?
